cfn-resources/encryption-at-rest/docs/README.md

```diff
@@ -0,0 +1,77 @@
+# MongoDB::Atlas::EncryptionAtRest
+
+This resource allows administrators to enable, disable, configure, and retrieve the configuration for Encryption at Rest.
```

Comment:
Update definition here to match with other readme file

Reply:
This issue is fixed